package.json

```diff
@@ -50,7 +53,7 @@
     "lint:css": "stylelint 'public/*.css'",
     "lint:js": "eslint .",
     "start": "node server/portal_server",
-    "test": "mocha",
+    "test": "mocha test/integration && mocha test/unit",
```

Comment:
Because the tests were refactored into directories, I think you need to tweak the "format" script now.

Probably something like this so it gets the nested test/**/*.js files:

```js
"format": "prettier '{frontend/src/,scripts/,server/,test/**/}*.js' 'public/*.css' --single-quote --write",
```